src/main/java/InfrastructureManager/NetworkStructure/Network.java

```diff
@@ -0,0 +1,182 @@
+    private List<DeviceLocationRelation> deviceLocationList;
+	
+	/**
+	* Default constructor for Network Class
+	*/
+	public Network() {
+		this.mapper = new ObjectMapper();
+		deviceList = new ArrayList<>();
+		connectionList = new ArrayList<>();
+		locationList = new ArrayList<>();
+		applicationList = new ArrayList<>();
+		applicationDeviceList = new ArrayList<>();
+		locationConnectionList= new ArrayList<>();
+		deviceLocationList = new ArrayList<>();
+	}
+
+	/**
+	 * Function to read JSON string and update network class
+	 * @param String
+	 */
+	public void loadNetwork(String readString) throws JsonProcessingException{
+		
+		//New object created as the called object cannot be accessed as a whole. Therefore new object is created to update the calling object members with values from json string
+		Network clonedObject = this.mapper.readValue(readString,Network.class);
+		this.deviceList = clonedObject.deviceList;
+		this.connectionList = clonedObject.connectionList;
+		this.locationList = clonedObject.locationList;
+		this.applicationList = clonedObject.applicationList;
+		this.applicationDeviceList = clonedObject.applicationDeviceList;
+		this.locationConnectionList = clonedObject.locationConnectionList;
+		this.deviceLocationList = clonedObject.deviceLocationList;
+	}
+	
+	/**
+	 * Function to save JSON string
+	 * @return String
+	 */
+	public String saveNetwork() throws JsonProcessingException{
+		String json = null;
+		try {
+			// convert network object to JSON
+			json = this.mapper.writeValueAsString(this);
+		} catch (Exception ex) {
+		    ex.printStackTrace();
+		}
+		return json;
+		
+	}
+	
+	/**
+	 * Getter function
+	 * @return deviceList
+	 */
+    public List<Device> getDeviceList() {
+        return deviceList;
+    }
+    
+	/**
+	 * Getter function
+	 * @return connectionList
+	 */
+    public List<Connection> getConnectionList() {
+        return connectionList;
+    }
+	
+	/**
+	 * Getter function
+	 * @return locationList
+	 */
+    public  List<Location> getLocationList() {
+        return locationList;
+    }
+    
+	/**
+	 * Getter function
+	 * @return applicationList
+	 */
+    public List<ApplicationInstance> getApplicationList() {
+        return applicationList;
+    }
+
+	/**
+	 * Getter function
+	 * @return applicationDeviceList
+	 */
+    public List<ApplicationInstanceDeviceRelation> getApplicationDeviceList() {
+        return applicationDeviceList;
+    }
+    
+	/**
+	 * Getter function
+	 * @return locationConnectionList
+	 */
+    public List<LocationConnectionRelation> getLocationConnectionList() {
+        return locationConnectionList;
+    }
+    
+	/**
+	 * Getter function
+	 * @return deviceLocationList
+	 */
+    public List<DeviceLocationRelation> getDeviceLocationList() {
+        return deviceLocationList;
+    }
+    
+	/**
+	 * Setter function
+	 * @param deviceList
+	 */
+    public  void setDeviceList(List<Device> deviceList) {

```

Comment:
setting the whole list is not recommended, instead, adding and removing a specific device should be implemented.